modules/virt-creating-a-primary-udn.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-primary-udn.adoc
+
+:_mod-docs-content-type: PROCEDURE                                
+[id="virt-creating-a-primary-udn_{context}"]                                  
+= Creating a primary namespace-scoped user-defined network
+
+You can create an isolated primary network in your project namespace. You must use the OVN-Kubernetes layer 2 topology and enable persistent IP address allocation in the user-defined network (UDN) configuration to ensure VM live migration support.
+
+.Prerequisites
```

Comment:
@phoracek do we need any sort of clarification / differentiation for a non-admin user ?

Reply:
I would leave it as is, not to overcomplicate the guide. As far as I'm concerned, the whole process of defining a network is for the cluster admin persona.